project.clj

```diff
@@ -0,0 +1,13 @@
+(defproject neo4clj "1.0.0-SNAPSHOT"
```

Comment:
I think the following is possible ?!?

Maybe give the project a group like `fullspectrum/neo4clj` but then remove the `full-spectrum.` from all namespaces because it is now possible to specify in the dependency instead:

```
;; projects.clj
:dependencies [[fullspectrum/neo4clj "1.0.0"]]

;; Any clojure file
(:require [neo4j.client :as client])
```

Btw. is it on purpose that you didn't name it neo4jclj (2 j's)?

Reply:
Ill test it out, and yes it is on purpose it is named without the second j